react/features/app-settings/components/styles.js

```diff
@@ -4,32 +4,49 @@ import {
     createStyleSheet
 } from '../../base/styles';
 
-const LABEL_TAB = 300;
-
 export const ANDROID_UNDERLINE_COLOR = 'transparent';
+export const CONTAINER_PADDING = 2 * BoxModel.padding;
+export const HEADER_COLOR = ColorPalette.blue;
+export const HEADER_PADDING = BoxModel.padding;
 
 /**
- * The styles of the React {@code Components} of the feature welcome including
- * {@code WelcomePage} and {@code BlankPage}.
+ * The styles of the React {@code Components} of the feature
+ * {@code AppSettings}.
  */
 export default createStyleSheet({
 
+    /**
+    * The back button style on Android platform.
+    */
+    backIconAndroid: {
```

Comment:
I think this could be simplified using `Platform.select`: https://facebook.github.io/react-native/docs/platform-specific-code.html